templates/site/test-button.html

```diff
@@ -0,0 +1,46 @@
+{% extends "layouts/main.html" %}
+{% set page_title = "Test Buttons" %}
+
+{% block content %}
+    
+    {% with med_action_button = true%}
+        {% set button_disabled = "" %}
+        {% set button_label = "Let's Do It" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+    
+    {% with med_action_button_alt = true%}
+        {% set button_disabled = "" %}
+        {% set button_label = "Let's Do It" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    {% with med_action_button_alt = true%}
+        {% set button_disabled = "disabled" %}
```

Comment:
If it's disabled then make this true and apply the appropriate classes in the partial. As it is right now, I could add any text here and it would be used as a new class.

Reply:
Changed disabled toggle to use true.